pkg/util/output.go

```diff
@@ -46,24 +46,34 @@ func Eprintf(format string, args ...interface{}) {
 	fmt.Fprintf(Stderr, format, args...)
 }
 
-// PrintYaml prints the object to yaml to Stdout
-func PrintYaml(obj interface{}) {
+// GetYaml returns the yaml representation of obj
+func GetYaml(obj interface{}) string {
```

Comment:
Instead of "GetYaml" which feels like we are "fetching a GET call to get a yaml value" we can call it `YamlMarshal()` or `ToYaml(...)`

Reply:
done